commit b22322bfd104949b93d1df22ff36689747917448
Author: Daniel Eklöf <daniel@ekloef.se>
Date:   Mon Nov 15 19:22:06 2021 +0100

    config: letter-spacing: add missing ‘return’
    
    The missing return caused us to fall through to the bottom of the
    function, where we ended with an error message:
    
      [main].letter-spacing: 0: not a valid option: letter-spacing
    
    Closes #795

diff --git a/config.c b/config.c
index 905f26d6..2f12e201 100644
--- a/config.c
+++ b/config.c
@@ -956,7 +956,7 @@ parse_section_main(struct context *ctx)
         return value_to_pt_or_px(ctx, &conf->line_height);
 
     else if (strcmp(key, "letter-spacing") == 0)
-        value_to_pt_or_px(ctx, &conf->letter_spacing);
+        return value_to_pt_or_px(ctx, &conf->letter_spacing);
 
     else if (strcmp(key, "horizontal-letter-offset") == 0)
         return value_to_pt_or_px(ctx, &conf->horizontal_letter_offset);
